.github/workflows/test_integration_cli.yml

```diff
@@ -43,6 +43,7 @@ on:
           - "3.8"
           - "3.9"
           - "3.10"
+          - "3.11"
```

Comment:
I thought 3.11 was not supported.

Reply:
It isn't supported yet, but I am leaving open the option to easily test it.